src/app/api/score/score.py

```diff
@@ -86,7 +86,11 @@ def search_score(node_id: uuid.UUID, resource_type: ResourceType) -> dict:
 
 
 def node_id_param(
-    *, node_id: uuid.UUID = Path(..., examples=COLLECTION_NAME_TO_ID)
+    *,
+    node_id: uuid.UUID = Path(
+        ...,
+        examples=create_examples(COLLECTION_NAME_TO_ID),
```

Comment:
should the root node be part of the examples like a the other examples?

Reply:
Yes, refactored